tools/cli/commands/create.js

```diff
@@ -48,13 +49,42 @@ function toUpperCamelCase(words) {
     return formatted;
 }
 
+/**
+ * Validate if a name is valid custom element name
+ * @param {string} name
+ * @returns {boolean}
+ */
+function isNameValid(name) {
+    const wordsBetweenHyphensRegExp = new RegExp(/(?<=[a-z]-)[a-z]+/);
+    const specialCharactersRegExp = new RegExp(/\?|!|<|>|@|#|\$|%|\^|&|\*|\(|\)|\+|\[|\]|~/);
+
+    if (specialCharactersRegExp.test(name)) {
+        console.error(`Custom element's name must not contain any of the following characters - ?  !  <  >  @  #  $  %  ^  &  *  (  )  +  [  ] ~`);
+        return false;
+    }

```

Comment:
```suggestion
    const specialCharactersRegExp = new RegExp(/\?|!|<|>|@|#|\$|%|\^|&|\*|\(|\)|\+|\[|\]|~/g);

    if (specialCharactersRegExp.test(name)) {
        console.error(`Custom element's name must not contain any of the following characters - ?  !  <  >  @  #  $  %  ^  &  *  (  )  +  [  ] ~. Current name uses ${name.match(specialCharactersRegExp).join(' ')}`);
        return false;
    }

```
Adding it as an option. TBH not sure if really needed, just more info for the user.